databaseutil/databaseutil.go

```diff
@@ -36,24 +36,21 @@ func InsertIntoUsersTable(
 	password []byte,
 	creationTime time.Time,
 ) error {
-	var row *sql.Row
-	{
-		sqlQuery := `
-			INSERT INTO users (display_name, email_address, password, creation_time)
-			VALUES ($1, $2, $3, $4)`
+	sqlQuery := `
```

Comment:
Started looking at this code due to: https://github.com/atmiguel/cerealnotes/pull/16#discussion_r178424005 and realized we shouldn't be using the query row here since it requires a row so i was able to fix the code by using query instead.

Reply:
what do you mean by "it requires a row"?